### PR TITLE
[13.0][ADD] sale_operating_unit_sequence

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module use to define sequence of Sale Orders by Operating Unit.
cc: @newtratip @kittiu 

This PR is waiting for https://github.com/OCA/operating-unit/pull/252, you can always use this for readability.
- [x] [13.0][MIG] migration of module sale_operating_unit

### CODE REVIEW

**Review by newtratip — COMMENTED** [review]
Functional Test :+1:  Wait another review.

File: sale_operating_unit_sequence/models/operating_unit.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class OperatingUnit(models.Model):
+    _inherit = "operating.unit"
+
+    so_sequence_id = fields.Many2one(
```
Comment (kittiu) [inline sale_operating_unit_sequence/models/operating_unit.py]: I would prefer `sale_sequence_id`

Comment (ps-tubtim) [reply sale_operating_unit_sequence/models/operating_unit.py]: I changed `so_sequence_id` to `sale_sequence_id`.

**Review by alan196 — APPROVED** [review]
Technical and Functional Review 👍

**Review by newtratip — APPROVED** [review]

**Pull request merged**